airbyte-webapp/src/components/connectorBuilder/YamlEditor/YamlEditor.tsx

```diff
@@ -1,26 +1,37 @@
 import { useMonaco } from "@monaco-editor/react";
 import { load, YAMLException } from "js-yaml";
+import isMatch from "lodash/isMatch";
 import { editor } from "monaco-editor/esm/vs/editor/editor.api";
-import { useEffect, useRef, useState } from "react";
-import { useDebounce, useLocalStorage } from "react-use";
+import { useEffect, useMemo, useRef, useState } from "react";
 
 import { CodeEditor } from "components/ui/CodeEditor";
 
 import { ConnectorManifest } from "core/request/ConnectorBuilderClient";
-import { useManifestTemplate } from "services/connectorBuilder/ConnectorBuilderApiService";
+import { useConfirmationModalService } from "hooks/services/ConfirmationModal";
 import { useConnectorBuilderState } from "services/connectorBuilder/ConnectorBuilderStateService";
 
+import { convertToManifest } from "../Builder/types";
```

Comment:
As these are used in both the builder and the yaml editor, should we pull them into the top level connector builder directory?